test/unittest/tests/block/test_block_api_write.cpp

```diff
@@ -0,0 +1,279 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * [block_api]: block_write.c
+ * The block manager performs file operations on specific blocks which eventually gets translated
```

Comment:
The word translated here is weird. Maybe: "The block manager writes to files in discrete chunks know as blocks. This set of test validates the write(), read() and write_size() APIs." Also does this really validate the read api? The only reason we think it does it because we use it to validate the read API but when did we actually test read on it's own?